### ISSUE TITLE
Can't upload

### ISSUE THREAD

**Issue by yustnip** [avocode/avocode#421]
IMPORTANT: Please try to describe how did it happen.
Can't upload a new PSD. Maybe it's because of slow internet but I don't know.

-----
App v.2.4.2-d618096
Source UID: 113072
```
Error: Invariant Violation: Dispatch.dispatch(...): Cannot dispatch in the middle of a dispatch.
    at invariant (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/flux/lib/invariant.js:42:15)
    at o.Dispatcher.dispatch (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:185:5)
    at o.handleViewAction (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/@avocode/core/lib/dispatcher.js:1:707)
    at t.module.exports.t.setDocument (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/@avocode/core/lib/active-document.js:1:1668)
    at t._fireNoActiveViewEvents (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:24433)
    at t._fireNoActiveViewEvents (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:999)
    at atom-workspace.<anonymous> (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:8427)
    at atom-workspace.handler (/home/yustnip/apps/Avocode/resources/app.asar/src/space-pen-extensions.js:110:34)
    at atom-workspace.jQuery.event.dispatch (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4681:9)
    at atom-workspace.elemData.handle (/home/yustnip/apps/Avocode/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4359:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/home/yustnip/apps/Avocode/resources/app.asar/src/command-registry.js:251:29)
    at CommandRegistry.handleCommandEvent (/home/yustnip/apps/Avocode/resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/home/yustnip/apps/Avocode/resources/app.asar/src/command-registry.js:166:19)
    at PaneContainer
```

**Comment by yustnip** [comment]
FYI: Uploaded this PSD from the 3d attempt.

**Comment by chazchumley** [comment]
Can upload Sketch file using Plugin.  WTF

**Comment by SkoebaSteve** [comment]
Any updates about this?

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.
